autoload/go/util.vim

```diff
@@ -85,6 +85,21 @@ function! go#util#Shelllist(arglist, ...)
     endtry
 endfunction
 
+" Returns the byte offset for line a:l and column a:c
+function! go#util#Offset(l, c)
```

Comment:
just rename the arguments to `line` and `col` and change the doc to reflect it: `Returns the byte offset for the given line and column`